Compilers/Windows/gcc.ey

```diff
@@ -8,18 +8,18 @@ Target-platform: Windows
 # Some info about it
 path:
 tcpath:
-make: mingw32-make.exe
-makeflags: SHELL=cmd.exe
+make: make
```

Comment:
This hasn't been changed in ages... why is it suddenly wrong, and are you certain it doesn't break users' current configurations?

Reply:
Again it's because we are going with msys2/mingw-w64 where it's just called make. Also you told me that ``SHELL=cmd.exe`` line doesn't do anything and could be removed.

Reply:
I know that this test harness is using MSys, but I'm not sure if the package we bundled for typical users will tolerate this. Have you tested this manually in your existing LGM setup?

Reply:
Yeah this works but what breaks it is removing the -I includes for /Additional just a few lines below this. If that's pulled from the Portable ZIP it will not work. Not sure what to do about that because the MSys2 setup doesn't have/use that folder.

Reply:
Commit 5c9febd4d0e39d1ff1ca5d8fb7c6c8b68abe7389 takes care of this. @fundies said we can just keep the includes for both and I went and tested a full rebuild of both the ZIP and the Msys2 setup. This should be good now.

Reply:
Sounds good. Just wanted to be sure you weren't breaking actual releases trying to get tests to pass.